### PR TITLE
Add 'ci' Rake task and travis configuration.

### PR DESCRIPTION
**Pull request by coblej**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: ruby
+rvm:
+  - 2.4.2
+before_script:
+ - "cp config/role_map.yml.sample config/role_map.yml"
+script: "bundle exec rake ci"
+jdk:
+  - oraclejdk8
+services:
+  - redis-server
```
Comment (dchandekstark) [inline .travis.yml]: Does it need this in the test environment?

Comment (coblej) [reply .travis.yml]: @dchandekstark Couldn't tell you for sure ... it's just how we did it with Scholrax

Comment (dchandekstark) [reply .travis.yml]: I don't know why you would need it if the ActiveJob adapter is `:inline`.

Comment (coblej) [reply .travis.yml]: @dchandekstark I think Hyrax uses Redis for things other than Resque jobs (e.g., notifications) but, assuming this passes Travis, I can resubmit with 'redis-server' removed and see if it still passes if you like.

**Pull request merged**